modules/nf-core/oatk/main.nf

```diff
@@ -0,0 +1,65 @@
+process OATK {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/oatk:1.0':
+        'biocontainers/oatk:1.0' }"
+
+    input:
+    tuple val(meta), path(reads)
+    tuple path(mito_hmm), path(mito_hmm_h3f), path(mito_hmm_h3i), path(mito_hmm_h3m), path(mito_hmm_h3p)
+    tuple path(pltd_hmm), path(pltd_hmm_h3f), path(pltd_hmm_h3i), path(pltd_hmm_h3m), path(pltd_hmm_h3p)
+    path(tmp)
```

Comment:
Usually tmp folders don't need to be staged. Is there a necessity for staging the tmp folder?

Reply:
Good point. tmp can be specified in args if required.